src/OrbitGl/TimelineTicksTest.cpp

```diff
@@ -26,4 +26,24 @@ TEST(TimelineTicks, GetMajorTicks) {
                                    1 * kNanosecondsPerMinute));
 }
 
+TEST(TimelineTicks, GetTimestampsNumDigitsPrecision) {
+  TimelineTicks timeline_ticks;
+
+  EXPECT_EQ(timeline_ticks.GetTimestampsNumDigitsPrecision(0, 20), 8);   // 10ns = 0.000'000'01s
+  EXPECT_EQ(timeline_ticks.GetTimestampsNumDigitsPrecision(0, 299), 7);  // 100ns = 0.000'000'1s
```

Comment:
I',m not sure if I fully understand this. Is this based on the 50ns scale? So GetMajorTicks returns [0ns, 50ns, 100ns, 150ns, 200ns, 250ns]? And since 50ns and 150ns require 7 decimal places that's the result?

Reply:
We don't need labels for MinorTicks and I didn't want to make the name of this method longer, but another good clarity reason to refactor this method.